docs/bigtable.txt

```diff
@@ -0,0 +1,72 @@
+[[bigtable]]
+Google Cloud Bigtable
+---------------------
+
+[.tss-center.tss-width-250]
+image:Cloud-Bigtable.svg[link="https://cloud.google.com/bigtable/"]
+
+[quote, 'https://cloud.google.com/bigtable/[Google Cloud Bigtable Homepage]']
+____
+Cloud Bigtable is Google's NoSQL Big Data database service. It's the same database that powers many core Google
+services, including Search, Analytics, Maps, and Gmail.
+
+Bigtable is designed to handle massive workloads at consistent low latency and high throughput, so it's a great choice
+for both operational and analytical applications, including IoT, user analytics, and financial data analysis.
+____
+
+Bigtable Setup
+~~~~~~~~~~~~~~
+
+Bigtable implements the HBase interface for all data access operations, and requires some extra binary dependencies and
+some specific configuration options to connect.
+
+Dependencies
+^^^^^^^^^^^^
+
+Bigtable connectivity relies on the cloud-bigtable-client library. This library is open-source and is
```

Comment:
Add code formatting for "cloud-bigtable-client" if it's an artifact name, or change it to proper caps if not.